python/lsst/afw/table/catalogMatches.py

```diff
@@ -170,3 +174,37 @@ def matchesFromCatalog(catalog, sourceSlotConfig=None):
         matches.append(ReferenceMatch(ref, src, cat[distKey]))
 
     return matches
+
+
+def copyAliasMapWithPrefix(inSchema, outSchema, prefix=""):
+    """Copy an alias map from one schema into another
+
+    This copies one alias map into another, optionally appending a
+    prefix to the name on both sides of the alias (the example use
+    case here is for the "match" catalog created by
+    `lsst.meas.astrom.denormalizeMatches` where prefixes "src_" and
+    "ref_" are added to the source and reference field entries,
+    respectively).
+
+    Parameters
+    ----------
+    inSchema : `lsst.afw.table.Schema`
+       The input schema whose `lsst.afw.table.AliasMap` is to be
+       copied to ``outSchema``.
+    outSchema : `lsst.afw.table.Schema`
+       The output schema into which the `lsst.afw.table.AliasMap`
+       from ``inSchema`` is to be copied (modified in place).
+    prefix : `str`, optional
+       An optional prefix to add to both sides of the alias names
```

Comment:
Here and in the description above, "both sides of the alias" is unclear. Maybe something like "... optionally appending a prefix to the name for both the input and output field entries"?

Reply:
I see your point, but I don't think referring to input/output really works (as that could be confused with the input/output schemas), so I reworded to:
```
An optional prefix to add to both the "from" and "to" names
of the alias (default is an empty string).
```